lighthouse-core/report/v2/report-styles.css

```diff
@@ -17,6 +17,7 @@
  :root {
   --text-font-family: '.SFNSDisplay-Regular', 'Helvetica Neue', 'Lucida Grande', sans-serif;
   --body-font-size: 13px;
+  --body-line-height: 20px;
```

Comment:
I always see and prefer line-height specified as unitless: https://www.google.com/search?q=line-height+unitless

seems a bit better.